misc/webrtc-utils/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT"
 name = "libp2p-webrtc-utils"
 repository = "https://github.com/libp2p/rust-libp2p"
 rust-version = { workspace = true }
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
Wouldnt `0.2.1` suffice since this is a fix?

Reply:
It's not a breaking change, no. We can go with a minor increment for sure

Reply:
Changed :+1: